test/test_genome.py

```diff
@@ -548,3 +548,72 @@ def test_mutate_does_not_reinitialize_parameters(genome_params, rng, mutation_ra
     assert genome._parameter_names_to_values["<p2>"] == pytest.approx(
         parameter_names_to_values_before["<p2>"]
     )
+
+
+def test_genome_reordering_empirically(rng):
+    # empirically test that reordering doesn't change the output function of a genome
+
+    pytest.importorskip("sympy")
+
+    # target: f(x_0,x_1) = x_0 ** 2 - x_1 + 1
+
+    genome_params = {
+        "n_inputs": 2,
+        "n_outputs": 1,
+        "n_columns": 10,
+        "n_rows": 1,
+        "levels_back": None,
+        "primitives": (cgp.Mul, cgp.Sub, cgp.Add, cgp.ConstantFloat),
+    }
+
+    genome = cgp.Genome(**genome_params)
+
+    dna_fixed = [
+        ID_INPUT_NODE,
+        ID_NON_CODING_GENE,
+        ID_NON_CODING_GENE,
+        ID_INPUT_NODE,
+        ID_NON_CODING_GENE,
+        ID_NON_CODING_GENE,
+        0,  # Mul -> outs x_0² (address 2)
```

Comment:
```suggestion
        0,  # Mul -> x_0^2 (address 2)
```
please use `^` instead of actual superscripts ;)